apps/examples/wc-angular12/package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "wc-angular12",
+    "version": "0.0.0",
+    "scripts": {
+        "ng": "ng",
+        "dev": "ng serve",
+        "build": "ng build",
+        "watch": "ng build --watch --configuration development"
+    },
+    "private": true,
+    "dependencies": {
+        "@angular/animations": "15.2.0",
+        "@angular/common": "15.2.0",
+        "@angular/compiler": "15.2.0",
+        "@angular/core": "15.2.0",
+        "@angular/forms": "15.2.0",
+        "@angular/platform-browser": "15.2.0",
+        "@angular/platform-browser-dynamic": "15.2.0",
+        "@angular/router": "15.2.0",
+        "@justeattakeaway/pie-button": "0.10.2",
```

Comment:
In complete contrast to my other comment, can you add a `^` to the pie-button dependency (just as that'll mean it'll always try and use latest minor version).

Reply:
😂 
I always set a task before i set up a PR to get rid of all the `^`s so 'Ash doesn't have to tell me to' 
I feel like I don't know who I am anymore 😁

Reply:
There's always an exception to the rule – and if there's not you should make one up just to confuse everyone 😆